circuit-benchmarks/src/bench_params.rs

```diff
@@ -0,0 +1 @@
+pub(crate) const DEGREE: usize = 19;
```

Comment:
is this change needed?

Reply:
fixed with https://github.com/scroll-tech/zkevm-circuits/pull/345/commits/89fafcd8034338adf8a5945433adb36c944748c7